Looks like the project doesn't compile on Linux

@gadgetci reported…
```bash
2016-01-04 21:15:33.890 [info] <0.32528.95>@gadget_utils:run_command:113 make[2]: Entering directory `/tmp/gadget/inaka/erlang-github/0001451-0942105-0402250/deps/barrel_jiffy/c_src'
make[2]: Nothing to be done for `all'.
make[2]: Leaving directory `/tmp/gadget/inaka/erlang-github/0001451-0942105-0402250/deps/barrel_jiffy/c_src'
make -f c_src/Makefile.erlang.mk

2016-01-04 21:15:33.893 [info] <0.32528.95>@gadget_utils:run_command:113 make[2]: Entering directory `/tmp/gadget/inaka/erlang-github/0001451-0942105-0402250/deps/barrel_jiffy'
cc -o priv/barrel_jiffy_drv.so c_src/decoder.o c_src/encoder.o c_src/jiffy.o c_src/utf8.o c_src/util.o  -L /usr/local/lib/erlang/lib/erl_interface-3.7.20/lib -lerl_interface -lei   -shared

2016-01-04 21:15:33.902 [info] <0.32528.95>@gadget_utils:run_command:113 c_src/decoder.o: file not recognized: File format not recognized
collect2: error: ld returned 1 exit status
make[2]: *** [priv/barrel_jiffy_drv.so] Error 1
make[2]: Leaving directory `/tmp/gadget/inaka/erlang-github/0001451-0942105-0402250/deps/barrel_jiffy'
make[1]: *** [pre-app] Error 2
make[1]: Leaving directory `/tmp/gadget/inaka/erlang-github/0001451-0942105-0402250/deps/barrel_jiffy'
make: *** [deps] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -1,12 +1,12 @@
 PROJECT = egithub
 
-DEPS = lager barrel_jiffy ibrowse
+DEPS = lager jiffy ibrowse
 TEST_DEPS = katana mixer meck
 SHELL_DEPS = sync
 BUILD_DEPS = hexer_mk
 
 dep_lager = git https://github.com/basho/lager.git 3.0.2
-dep_barrel_jiffy = hex 0.14.4
+dep_jiffy = git https://github.com/davisp/jiffy.git 0.14.5
 dep_ibrowse = hex 4.2.2
 dep_katana = hex 0.2.18
 dep_mixer = git https://github.com/inaka/mixer.git 0.1.4

--- a/src/egithub_webhook.erl
+++ b/src/egithub_webhook.erl
@@ -45,7 +45,7 @@ event(Module, Cred, #{headers := Headers, body := Body}) ->
     <<"ping">> -> ok;
     <<"pull_request">> ->
       EventData = egithub_json:decode(Body),
-      case handle_pull_request(Module, Cred, EventData) of
+      case do_handle_pull_request(Module, Cred, EventData) of
         clean -> ok;
         with_warnings -> ok;
         {error, Error} -> {error, Error}
@@ -72,7 +72,7 @@ event(Module, StatusCred, ToolName, Context, CommentsCred, Request) ->
     <<"pull_request">> ->
       EventData = egithub_json:decode(Body),
       set_status(pending, StatusCred, ToolName, Context, EventData),
-      try handle_pull_request(Module, CommentsCred, EventData) of
+      try do_handle_pull_request(Module, CommentsCred, EventData) of
         clean ->
           set_status(success, StatusCred, ToolName, Context, EventData),
           ok;
@@ -143,7 +143,7 @@ normalize_state(State) -> State.
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%% Events
 
-handle_pull_request(Module, Cred,
+do_handle_pull_request(Module, Cred,
       #{<<"number">> := PR, <<"repository">> := Repository} = Data) ->
   Repo = binary_to_list(maps:get(<<"full_name">>, Repository)),
   {ok, GithubFiles} = egithub:pull_req_files(Cred, Repo, PR),

